R/utils.R

```diff
@@ -165,7 +165,8 @@ available_port <- function(port) {
 # 403 for HEAD requests. See
 # https://stat.ethz.ch/pipermail/r-devel/2016-June/072852.html
 download <- function(url, destfile, mode = "w") {
-  if (getRversion() == "3.3.0" || getRversion() == "3.3.1") {
+  if (getRversion() == "3.3.0" || getRversion() == "3.3.1" ||
```

Comment:
Do you mind changing it to just `getRversion() >= "3.3.0"`? I have a feeling that the issue isn't going to be fixed in R any time soon.